android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/manager/FetchDownloadManager.kt

```diff
@@ -59,6 +62,8 @@ class FetchDownloadManager<T : AbstractFetchDownloadService>(
     @VisibleForTesting
     internal fun getSDKVersion() = SDK_INT
 
+    internal fun getTargetSDKVersion() = applicationContext.applicationInfo.targetSdkVersion
```

Comment:
Think we can remove this as we will target 33 and above and this will always be true

Reply:
Done.